### ISSUE TITLE
question about padding

### ISSUE THREAD

**Issue by hiwfwang** [nnzhan/Graph-WaveNet#19]
Thanks for your wonderful code! I am confused that why you use  nn.functional.pad() on training and eval input but not on test input. Can you explain the reason?

**Comment by moghadas76** [comment]
This is also my question

**Comment by razvangabrielc** [comment]
I'm wondering the same. It would be great if someone could help us out.

**Comment by chentingyang** [comment]
plus one

**Comment by xingzhinie** [comment]
I had tried it, the result of removing the pad operation will  not be affected.
